integration_tests/snaps/multimple-main-go-packages/main1/main.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
"multimple?" /me runs to see if you made the same typo in the integration test so I can point it out.